_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-slate
```

Comment:
It is not, and it's not supposed to be in this branch. It was however reverted with the later commit d1584e2. Should I rebase to remove it and create a new PR so it doesnt turn up in the master git logs?